external/netfx/netfx.depproj

```diff
@@ -27,10 +27,14 @@
   <ItemGroup>
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System" />
+    <TargetingPackReference Include="System.ComponentModel.Composition" />
```

Comment:
These need to go in shims.proj.

Please don't add filtering back to this project unless you do a full BuildAllConfigurations=true build